flagscale/train/train_qwen2_5_vl.py

```diff
@@ -14,25 +14,21 @@
 # limitations under the License.
 
 
+import logging
 import os
 import sys
-import logging
-from functools import partial
+
+from argparse import Namespace
 from copy import deepcopy
+from functools import partial
 from typing import List, Optional, Tuple, Union
```

Comment:
Imports from the builtin libraries can be grouped together in alphabetic order.
There doesn't seem a need to have two import groups for these libs.